From: Eric Dorland <eric@debian.org>
Date: Sun, 10 Aug 2008 02:30:48 -0400
Subject: Our name should be Iceweasel, not Firefox.

This is used early in startup in some key places (like the main window
group).
---
 browser/app/application.ini |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/browser/app/application.ini b/browser/app/application.ini
index b866f47..ca98d67 100644
--- a/browser/app/application.ini
+++ b/browser/app/application.ini
@@ -38,7 +38,7 @@
 #filter substitution
 [App]
 Vendor=Mozilla
-Name=Firefox
+Name=Iceweasel
 Version=@APP_VERSION@
 BuildID=@GRE_BUILDID@
 #ifdef MOZ_SOURCE_REPO
